dev/devicelab/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 
 dependencies:
   args: ^0.13.4
+  image:
```

Comment:
In most other places we're more specific about the version. How about `^1.1.27`?